cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -3,16 +3,17 @@ NAME
                         definition
 
 SYNOPSIS
-       bal grpc [--input <proto-file-path>] [--output <path>]
-                        [--mode] (client | service | proxy)
+       bal grpc [--input] <proto-file-path> [--output] <path>
+                        [--mode] (client | service)
+                        [--proto_path] <path>
 
 DESCRIPTION
        Generate the Ballerina gRPC client/service sources for a given
        gRPC protocol buffer (Protobuf) definition.
 
 OPTIONS
-       --input <proto-file-path>
-           Path of the input `.proto` file.
+       --input <path>
+           Path to a .proto file or a directory containing multiple .proto files.
```

Comment:
```suggestion
           Path to a '.proto' file or a directory containing multiple '.proto' files.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/0ba70265ce04e785ee7394104e66b7355e09ef49